### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by LefterisJP**
- Adjusting the coding style for consistency after the talk we had

Depends on #30

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,14 @@
...
 For details on this project, please see the Ethereum wiki:
 https://github.com/ethereum/wiki/wiki/Ethash
+
+### Coding Style
+
+The main thing above all is code consistency.
+
+- Tabs for indentation. A tab is 4 spaces
+- Try to stick to the [K&R](http://en.wikipedia.org/wiki/Indent_style#K.26R_style),
+  especially for the C code.
```
Comment (gavofyork) [inline README.md]: only for the C code -> C++ should follow our C++ Coding Standards to the letter.

File: src/libethash-cl/ethash_cl_miner.cpp [hunk 2]
```diff
@@ -56,10 +56,10 @@ ethash_cl_miner::ethash_cl_miner()
...
-       {
-               m_queue.finish();
-       }
+	if (m_queue())
+	{
+		m_queue.finish();
+	}
 }
 
 bool ethash_cl_miner::init(ethash_params const& params, ethash_h256_t const *seed, unsigned workgroup_size)
```
Comment (gavofyork) [inline src/libethash-cl/ethash_cl_miner.cpp]: for param decls & naming, let's stick to C++ coding standards.

**Comment by LefterisJP** [comment]
rebased

**Comment by LefterisJP** [comment]
rebased again ....

**Pull request merged**